Firestore/Example/Tests/Integration/FSTDatastoreTests.mm

```diff
@@ -144,10 +144,13 @@ - (void)applySuccessfulWriteWithResult:(const MutationBatchResult &)batchResult
 }
 
 - (void)rejectFailedWriteWithBatchID:(BatchId)batchID error:(NSError *)error {
+  (void)batchID;
```

Comment:
It appears that this method is only being called in `HandleRejectedWrite`, and it's also "not implemented". 

If this method is never going to be implemented for tests, why do we bother to convert the `Status` into `error.ToNSError()` when passing it into `rejectFailedWriteWithBatchID`, instead of having the method take in `Status`?